services/coach_manager.js

```diff
@@ -669,4 +669,25 @@ exports.process_pay_reminder = async (input_data) => {
 
 exports.post_process_pay_reminder = async (resp) => {
   resp.status(200).send({ status: "success", message: "payment reminder sent successfully", data: {} })
+}
+
+exports.pre_process_get_enrolled_player_count_by_date = async (req) => {
+  return {coach_id:req.user.coach_id,date:req.body.date}
+}
+
+exports.process_get_enrolled_player_count_by_date = async (input_data) => {
+  const {coach_id,date} = input_data
+  const player_count = await models.Enrollment.count({
+    where: {coach_id: coach_id,
+            status: "active",
```

Comment:
discuss with vivek regarding status whether we have to show all the enrollment request or success enrollment count